### DISCUSSION TITLE
Demos are not working in next-urql package

### DISCUSSION THREAD

**Discussion by sunpietro** [urql-graphql/urql#992] [Q&A]
I'm trying to run demos inside the `next-urql` package, but they are all failing to run.

## System config
1. MacOS 10.15.6
2. NodeJS 12.18.0
3. Yarn 1.22.5
4. next-urql package version - `1.1.0`, and the latest `main`

## Steps to reproduce
1. Install deps in any demo directory, for instance: `/packages/next-urql/examples/4-with-navigation` - with a command `yarn`,
2. Run another command `yarn start`,
3. Notice the error:
```
[ error ] /.../.../Projects/urql/packages/next-urql/dist/next-urql.es.js
Module not found: Can't resolve 'urql' in '/.../.../Projects/urql/packages/next-urql/dist'
```

Screenshot:
![Screenshot 2020-09-20 at 07 40 32](https://user-images.githubusercontent.com/1736658/93695261-0087b480-fb15-11ea-9fdf-061c5a475cc0.jpg)

## Expected behaviour
Demos should be working as expected.

**Comment by JoviDeCroock** [comment]
You'll have to run `yarn build` on the urql root, this because `next-urql` relies on the built `urql` package. We'll need to find a way to simplify this

**Comment by sunpietro** [comment]
Thanks. It's working. Anyway, it looks like the Pokemon DB is not reachable.
